scripts/docker/runit/arm_user_files_setup.sh

```diff
@@ -67,5 +67,5 @@ chown arm:arm /etc/abcde.conf /etc/arm/config/abcde.conf
 
 # symlink $ARM_HOME/Music to $ARM_HOME/music because the config for abcde doesn't match the docker compose docs
 # separate rm and ln commands because "ln -sf" does the wrong thing if dest is a symlink to a directory
-rm -f $ARM_HOME/music
+rm -rf $ARM_HOME/music
```

Comment:
Careful here.  I worry that if someone has their actual music folder mapped to this location, it will remove all their music upon launch.

Reply:
agree, didn't think of that use case. Will work through another fix to mitigate that potential case.

Reply:
I wouldn't rm/mv any of the users folders. It would be simpler to just wipe the system links by using this instead

```
# Test if abcde.conf is a hyperlink, if so remove it
if [ -h /etc/arm/config/abcde.conf ]; then
  echo "Old hyper link exists removing!"
  unlink /etc/arm/config/abcde.conf
fi
# check if abcde is in config main location - only copy if it doesnt exist
if ! [ -f /etc/arm/config/abcde.conf ]; then
  echo "abcde.conf doesnt exist"
  cp /opt/arm/setup/.abcde.conf /etc/arm/config/abcde.conf
  chown arm:arm /etc/arm/config/abcde.conf
fi
# The system link to the fake default file -not really needed but as a precaution to the -C variable being blank
if ! [ -h /etc/abcde.conf ]; then
  echo "/etc/abcde.conf link doesnt exist"
  ln -sf /etc/arm/config/abcde.conf /etc/abcde.conf
fi

```

and then change from
```
# OUTPUTDIR="$home/music/"
OUTPUTDIR="/home/arm/music/"
```
to 
```
# OUTPUTDIR="$home/Music/"
OUTPUTDIR="/home/arm/Music/"
```
Inside arm/setup/.abcde.conf

This would save on risking removing the music folder, and would put the onus on the end user on moving the old folder files to the new location. It would make it a breaking change, but its safer than wiping the users old rips.

Reply:
Looks good, will add those checks

I have made changes to the docker and other setup scripts to only use music not Music, havent pushed that change yet. Will add your changes and check it all works